src/tests/ie_test_utils/common_test_utils/crash_handler.hpp

```diff
@@ -15,10 +15,16 @@ namespace CommonTestUtils {
 
 extern jmp_buf env;
 
+enum JMP_STATUS { ok = 0, anyError = 1, alarmErr = 2 };
+
 class CrashHandler {
+private:
+    static int MAX_TEST_WORK_TIME;
 public:
     CrashHandler();
```

Comment:
To initialize timeout in Constructor using default value?

Reply:
done